_posts/2019-11-12-docker-rename-to-container.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Pulp docker plugin will be renamed to Pulp container plugin in Pulp3
+author: Ina Panova
+tags:
+  - docker
+  - container
+---
+
+With Pulp 3 GA pulp_container plugin will come along.
```

Comment:
s/pulp_container/the pulp_container/ ? Ask natives though :)

Reply:
I'd ask to check articles everywhere :)

Reply:
`the pulp_container plugin` is more correct but `pulp_container plugin` also works. If you refer to pulp_container alone, don't use `the`.